hydra-cluster/src/Hydra/Cluster/Scenarios.hs

```diff
@@ -12,15 +12,55 @@ import Data.Aeson (object, (.=))
 import Data.Aeson.Lens (key, _JSON)
 import qualified Data.Set as Set
 import Hydra.Cardano.Api (Lovelace, TxId, selectLovelace)
-import Hydra.Cluster.Faucet (Marked (Fuel), queryMarkedUTxO, seedFromFaucet)
-import Hydra.Cluster.Fixture (Actor (Alice), actorName, alice, aliceSk)
+import Hydra.Cluster.Faucet (Marked (Fuel), queryMarkedUTxO, seedFromFaucet, seedFromFaucet_)
+import Hydra.Cluster.Fixture (Actor (..), actorName, alice, aliceSk, aliceVk, bob, bobSk, bobVk)
 import Hydra.Cluster.Util (chainConfigFor, keysFor)
 import Hydra.Ledger (IsTx (balance))
 import Hydra.Ledger.Cardano (Tx)
 import Hydra.Logging (Tracer, traceWith)
 import Hydra.Options (networkId, startChainFrom)
 import HydraNode (EndToEndLog (..), input, output, send, waitFor, waitMatch, withHydraNode)
 
+restartedNodeCanObserveCommitTx :: Tracer IO EndToEndLog -> FilePath -> RunningNode -> TxId -> IO ()
+restartedNodeCanObserveCommitTx tracer workDir cardanoNode hydraScriptsTxId = do
+  let clients = [Alice, Bob]
+  -- forM_ clients (\actor -> refuelIfNeeded tracer cardanoNode actor 100_000_000)
+  [(aliceCardanoVk, _), (bobCardanoVk, _)] <- forM clients keysFor
+  seedFromFaucet_ cardanoNode aliceCardanoVk 100_000_000 Fuel (contramap FromFaucet tracer)
+  seedFromFaucet_ cardanoNode bobCardanoVk 100_000_000 Fuel (contramap FromFaucet tracer)
+
+  aliceChainConfig <-
+    chainConfigFor Alice workDir nodeSocket [Bob]
+      -- we delibelately do not start from a chain point here to highlight the
+      -- need for persistence
```

Comment:
This is actually not so important here. It should work either way (with a fresh temp dir).

Could: remove this comment and not include `startChainFrom = Nothing` in the test scope. (Also below)